teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initializate(age, specialization, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services
```

Comment:
Please use `?` at the end of the method which returns a bool. So the method name should be `can_use_services`.

Reply:
done